src/engine/video/texture_controller.cpp

```diff
@@ -291,69 +236,6 @@ void TextureController::_DeleteTexture(GLuint tex_id)
         _last_tex_id = INVALID_TEXTURE_ID;
 }
 
-bool TextureController::_SaveTempTextures()
-{
-    bool success = true;
-
-    // Create the temporary directory.
-    std::string path = vt_utils::GetUserDataPath() + DIRECTORY_TEMPORARY;
-    if (!vt_utils::DoesFileExist(path)) {
-        if (!vt_utils::MakeDirectory(path)) {
-            success = false;
-            IF_PRINT_WARNING(VIDEO_DEBUG) << "Unable to create the temporary directory: " << path << std::endl;
-        }
-    }
-
-    // Create the temporary texture directory.
-    path = vt_utils::GetUserDataPath() + DIRECTORY_TEMPORARY_TEXTURE;
```

Comment:
Maybe DIRECTORY_TEMPORARY_TEXTURE and DIRECTORY_TEMPORARY can be removed as well?